### ISSUE TITLE
window.event and IFRAMEs

### ISSUE THREAD

**Issue by cvazac** [whatwg/dom#735]
Imagine the following code, that is **executed from inside a same-origin `IFRAME`**:
```
top.addEventListener('load', function(e) {
  console.assert(typeof window.event === 'undefined')
  console.assert(typeof top.event === e)
})
```

That asserts that the event object should be retrievable from the same window where the target node (`top`) lives. 

These asserts pass in: Chrome 70, Edge 18, Safari 10-12, and IE 11
These asserts fail in: Chrome 71+, Firefox 63-65 (after changing `dom.window.event.enabled`), IE 10

There's some relevant discussion in the [Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1527954).

The [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=932154) I filed was folded into [another](https://bugs.chromium.org/p/chromium/issues/detail?id=664676), which is still open.

What should happen? On which global object should `event` be available? If the handler is bound to a node that moves between contexts, do we consider "bind time" or "callback time"?

Demo can be found [here](https://cvazac.netlify.com/window-dot-event/).

**Comment by annevk** [comment]
We decided this in https://github.com/whatwg/dom/issues/334#issuecomment-379707829. There might be some alternative model workable, but we did go through a fair amount of those already and they all had shortcomings.

cc @yuki3 @domenic @cvrebert @miketaylr @smaug---- @cdumez

**Comment by cvazac** [comment]
Just so I understand those conclusions: `event` will be on the global of the associated listener at bind-time, and will not change even if the target is moved between realms after the listener is bound. 

Do I have that right?

**Comment by annevk** [comment]
I'm not sure what bind-time is. Each function in JavaScript has a global (not explicitly acknowledged by ECMAScript currently, but Web IDL monkeypatches it and it'll be a more formal thing in due course) and that global is the one that's used.

**Comment by cvazac** [comment]
My mistake. :)

Let me try again:
`event` will be on the global of the associated listener function, regardless of where the target lives (or lived)

**Comment by annevk** [comment]
Aye.

**Comment by domenic** [comment]
Looks like specs and most implementations are behaving as desired here, so I'll close this.
